bika/lims/browser/reports/productivity_analysesperclient.py

```diff
@@ -115,7 +115,8 @@ def __call__(self):
             c_proxies = pc(portal_type="Client", sort_on='sortable_title')
 
         for client in c_proxies:
-            query['getClientUID'] = client.UID
+            # list of projects of this client
+            query['getClientUID'] = [project.getObject().UID() for project in client.getObject().getProjects()]
```

Comment:
Analyses per sample type report
Traceback (innermost last):
Module bika.lims.browser.reports.productivity_analysespersampletype, line 47, in __call__
AttributeError: getProjects